eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,62 @@
+param (
+  [Parameter(mandatory = $true)]
+  $artifactName,
+  [Parameter(mandatory = $true)]
+  $packageName
+)
+
+# TODO: delete testing comments below
+#'@azure/eventhubs-checkpointstore-table' alpha, no beta, no GA
+#'@azure/openai' alpha, beta, no GA
+#'@azure/video-indexer-widgets' no alpha, no beta, GA
+#'@azure/template' alpha, beta, GA
+#$pkgProps.PackageId
+
+$ErrorActionPreference = 'Stop'
+$PSNativeCommandUseErrorActionPreference = $true
+
+. (Join-Path $PSScriptRoot "../common/scripts/common.ps1")
+
+$packageProperties = Get-PkgProperties -PackageName $packageName
```

Comment:
We will need to decide if this validation or a variation of it is interesting for running outside of the JS sdk repo context. For publishing packages in other context. If we do want to use it in other contexts, which I suspect we might, then we cannot use this Get-PkgProperties function because it only works in the context of the JS repo. We might instead need to use some variation of https://github.com/Azure/azure-sdk-for-js/blob/main/eng/scripts/Language-Settings.ps1#L83 Get-javascript-PackageInfoFromPackageFile which reads the data from the package itself.